junitparser/junitparser.py

```diff
@@ -231,7 +231,6 @@ class JUnitXml(Element):
         tests: total number of tests
         failures: number of failed cases
         errors: number of cases with errors
-        skipped: number of skipped cases
```

Comment:
I am not sure about removing this, I'll have to check some specs. At least this has to be made backward-compatible or released under a major version.

Reply:
I understand. I linked the specs I could find in the commit message / PR description.